### ISSUE TITLE
Cannot find the script wp-phptidy.php

### ISSUE THREAD

**Issue by Ingram-AV-98** [welovewordpress/SublimePhpTidy#34]
Hi,

I'm using ST2 on Ubuntu 14.04 LTS as a Virtualbox guest and I keep being told that "PhpTidy cannot find the script at /home/michael/.config/sublime-text-2/Packages/PhpTidy/wp-phptidy.php.".

I've checked and the file is in the folder, I tried another suggestion:

<code>chmod +x phptidy.php wp-phptidy.php</code>

This didn't seem to help the situation, not sure what else I can do :(
I've restarted ST2 a few times already too with no joy.

**Comment by Geczy** [comment]
Does the folder "PhpTidy" exist?

**Comment by jancbeck** [comment]
Thanks @Geczy, this worked for me.

**Comment by Ingram-AV-98** [comment]
Yeah the "PhpTidy" folder is there with 14 files inside, including both "phptidy.php" and "wp-phptidy.php".

I have no problems on Win8.1, just my virtual Ubuntu.

**Comment by anmil** [comment]
I had this issue too. I ended up needing to install php in order to run it.

**Comment by jarednova** [comment]
I had the same issue. Turns out that I had tweaked the MAMP versions of PHP (which OSX was relying on). After setting them correctly, all was well

**Comment by nicolasgalvez** [comment]
I had the same issue, even though I could see the file. I'm on linux mint with xampp. I looked at the php_tidy.py file and changed line 30 to phppath = '/opt/lampp/bin/php'.

That seemed to work for me. I'm not sure if there's a better way to do it though.
